src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -75,6 +79,7 @@
                 :on-press #(>evt [:contact-requests.ui/decline-request id])}
      :button-2 {:label    (i18n/label :t/accept)
                 :type     :success
+                :override-background-color colors/success-60
```

Comment:
Could you please apply formatting again in the files changed by this PR? It's just that some map keys are not aligned properly according to the Clojure Style Guide. I think VS Code's Calva plugin works out of the box, as well as Emacs with the command `clojure-align`.

Reply:
done